src/libs/antares/study/include/antares/study/sets.hxx

```diff
@@ -21,78 +21,11 @@
 #ifndef __ANTARES_LIBS_STUDY_SETS_HXX__
 #define __ANTARES_LIBS_STUDY_SETS_HXX__
 
-namespace Antares
+namespace Antares::Data
 {
-namespace Data
```

Comment:
**Sets** is no longer a class template. So many method were moved to a new **sets.cpp** source file.
There are methods left : they have an extra template parameter. 
These template parameters may be removed very soon, as they are useless : used with only one effective type.